zephyr/module.yml

```diff
@@ -0,0 +1,2 @@
+build:
+  cmake: zephyr
```

Comment:
Please add:
```
  kconfig-ext: True
```
so that the Kconfig file follows the new pattern.
See more here: https://docs.zephyrproject.org/latest/guides/modules.html#module-integration-files-in-zephyr

You may also consider to move the CMake file itself into the Zephyr repo, that way there are fewer Zephyr specific patches in this repo.